smartsim/_core/_cli/build.py

```diff
@@ -66,8 +66,15 @@ def __init__(self):
             type=str,
             help="Path to custom libtensorflow directory (ONLY USED IF NEEDED)",
         )
+        parser.add_argument(
```

Comment:
Instead of making it so that we either install Redis or KeyDB, it would be good to be able to build KeyDB as an additional option. We can then switch out which one they use by setting the REDIS_PATH variables and so more easily swap out between the two for development, testing, and benchmarking.

Reply:
@Spartee I get what you're saying and within a `modules` system it's easy enough to do. Thinking about this a little more, the origin of this comment is more along the lines of what we think the expected behavior should be if a user does something like:
```
smart build
```
and then later (weeks/months) does
```
smart build --keydb
```
and for whatever reason switching to KeyDB degrades performance in their application (e.g. they haven't futzed with the settings enough). How do they switch back? Right now either the user has to know/remember that they can set `REDIS_PATH` or they can `smart clobber` and build from scratch.

What I think the question boils down to is: are we sure enough that KeyDB is going to be a drop-in improvement over `redis`?

Reply:
@ashao In the case of a user running `smart build --keydb` weeks/months later, since the implementation of #165, the user will no longer have to know/remember what to do as they will be informed by the cli on how to proceed. This probably doesn't address the entirety of your comment, but hopefully it at least partially does.